test/Base_Tests/src/Runtime/Managed_Resource_Spec.enso

```diff
@@ -57,6 +58,10 @@ add_specs suite_builder = suite_builder.group "Managed_Resource" group_builder->
         r_3 = Panic.recover Any <| Managed_Resource.bracket 42 (_-> Nothing) (_-> Panic.throw "action")
         r_3.catch . should_equal "action"
 
+    group_builder.specify "allocate lots of resources at once" <|
```

Comment:
This test was disabled by 
- #10642

Attempt to diagnose what the problem was is at